build.yaml

```diff
@@ -900,6 +900,22 @@ libs:
   - grpc++_codegen
   secure: check
   vs_project_guid: '{C187A093-A0FE-489D-A40A-6E33DE0F9FEB}'
+- name: grpc++_reflection
+  build: all
+  language: c++
+  public_headers:
+  - extensions/include/grpc++/impl/proto_server_reflection_plugin.h
```

Comment:
Now I see the Makefile.template change and the installation should be fine. But I still think we should consider the file location.